src/main/java/dev/ebyrdeu/movie_jakarta/dto/MovieDto.java

```diff
@@ -1,8 +1,10 @@
 package dev.ebyrdeu.movie_jakarta.dto;
 
 import dev.ebyrdeu.movie_jakarta.entity.Movie;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.Positive;
 
-public record MovieDto(String title, int releaseYear, String director) {
+public record MovieDto(@NotEmpty String title, @Positive int releaseYear, @NotEmpty String director) {
```

Comment:
```suggestion
public record MovieDto(@NotBlank String title, @Positive int releaseYear, @NotBlank String director) {
```

We should use the annotation NotBlank to neglect all input containing only whitespaces. Since whitespace is a char, a String of " " would be valid if we usen NotEmpty.

Reply:
Thanks, will change to suggested annotations